src/style.css

```
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+  padding: 2px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+}
+
+.stars--0:nth-child(1),
+.stars--1:nth-child(2),
+.stars--2:nth-child(3),
+.stars--3:nth-child(4),
+.stars--4:nth-child(5),
+.stars--5:nth-child(6) {
+  margin-left: -4px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
```

check you tests before review. You have failed tests

```suggestion
.stars--1:nth-child(1),
.stars--2:nth-child(-n+2),
.stars--3:nth-child(-n+3),
.stars--4:nth-child(-n+4),
.stars--5:nth-child(-n+5) {
```